tests/unit/specs/scripts/common.spec.js

```diff
@@ -1,4 +1,11 @@
-import { coinsToObject, calculateShares, toMicroDenom, sleep } from "scripts/common"
+import {
+  coinsToObject,
+  calculateShares,
+  toMicroDenom,
+  sleep
+} from "scripts/common"
+
+jest.useFakeTimers()
```

Comment:
this should go into a test and at the end of the test it should be removed by using `jest.useRealTimers()` just so we avoid side effects

Reply:
Fixed